k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -16,13 +16,26 @@ function usage() {
   echo "                                     default file path is ${solution_yaml}."
   echo "    -n|--nodename NODENAME: collects logs from pods running only on given node".
   echo "                            collects logs from all the nodes by default."
+  echo "    --duration DURATION : dutaion for which logs should be collected"
+  echo "    --size_limit SIZE : max log size "
```

Comment:
max size limit for support bundle to be generated. Or something like this.

Reply:
done